man/docker-network-create.1.md

```diff
@@ -14,6 +14,7 @@ docker-network-create - create a new network
 [**--ip-range**=*[]*]
 [**--ipam-driver**=*default*]
 [**-o**|**--opt**=*map[]*]
+[**--ipam-opt**=*map[]*]
```

Comment:
this list should be ordered alphabetically. So this option needs to go after `ipam-driver`.